src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,31 @@
+import { useNavigate } from "react-router-dom";
+import "../../styles/Navbar.scss";
+import logo from "../../assets/logo.png";
+import profile from "../../assets/profile.png";
+
+const Navbar = () => {
+  const nav = useNavigate();
+
+  return (
+    <nav className="Navbar">
+        <div></div>
+        <img
+          className="Navbar-logo"
+          onClick={() => {
+            nav("/list");
+          }}
+          src={logo}
+          alt="로고"
+        />
+        <img
+          className="Navbar-profile"
+          onClick={() => nav("/mypage")}
+          src={profile}
+          alt="프로필"
+        />
```

Comment:
[Link, Navigate, useNavigate는 언제 쓰는게 좋을까?](https://www.codeit.kr/topics/building-a-website-with-react/lessons/5119)

`useNavigate`는 특정한 코드의 실행이 끝나고 페이지를 이동시킬 때 사용하고,
`Link`는 사용자가 클릭해서 페이지를 이동할 때 사용합니다.

현재는 사용자가 프로필 이미지를 클릭하면 mypage로 이동하기 때문에
`Link`를 사용하는 것이 더 적절해 보여요 이렇게 변경할 수 있을 것 같습니다!

```suggestion
<Link to="/mypage">
        <img
          className="Navbar-profile"
          src={profile}
          alt="프로필"
        />
 </Link>
```